personalities/ct-static-api/storage.go

```diff
@@ -21,26 +21,28 @@ import (
 	"github.com/transparency-dev/trillian-tessera/ctonly"
 )
 
+// Storage provides all the storage primitives necessary to write to a ct-static-api log.
 type Storage interface {
 	// 	Add assign an index to the provided Entry, stages the entry for integration, and return it the assigned index.
 	Add(context.Context, *ctonly.Entry) (uint64, error)
 }
 
-// CTStorage implements Storage
+// CTStorage implements Storage.
 type CTStorage struct {
-	StoreData func(context.Context, *ctonly.Entry) (uint64, error)
+	storeData func(context.Context, *ctonly.Entry) (uint64, error)
 	// TODO(phboneff): add storeExtraData
 	// TODO(phboneff): add dedupe
 }
 
-// NewStorage
+// NewCTStorage instantiates a CTStorage object.
 func NewCTSTorage(logStorage tessera.Storage) (*CTStorage, error) {
 	ctStorage := new(CTStorage)
-	ctStorage.StoreData = tessera.NewCertificateTransparencySequencedWriter(logStorage)
+	ctStorage.storeData = tessera.NewCertificateTransparencySequencedWriter(logStorage)
```

Comment:
nit: the `new` is a bit unusual, was there a driver for doing it that way, or could it be more idiomatic like below?
```
ctStorage := &CTStorage{storeData: tessera.New...}
return ctStorage, nil
```